packages-internal/scripts/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "2.0.12",
   "author": "MUI Team",
   "description": "Utilities supporting MUI libraries build and docs generation. This is an internal package not meant for general use.",
-  "main": "build/index.js",
```

Comment:
Doesn't have to part of this scope, but maybe we could do some "package.json linting" in our build step? To error or warn on things like this?

Reply:
Noted. Will add in `code-infra`.